src/vm/callhelpers.h

```diff
@@ -376,7 +376,6 @@ class MethodDescCallSite
             WRAPPER_NO_CONTRACT;                                                                \
             {                                                                                   \
                 GCX_FORBID();  /* arg array is not protected */                                 \
```

Comment:
You can delete this line and scope too.

Reply:
Thank you, will do as a separate change, there are several places like this, will try to clean them as well.

Reply:
It should be done in the same PR. The only reason for the extra scope and `GCX_FORBID` here is the call you are deleting.

Reply:
I do not understand, there are several palces like:
https://github.com/dotnet/coreclr/blob/bdb995987178231ba541f22143cb3cab56309daa/src/vm/callhelpers.h#L363-L369
where we have this extra scope and `GCX_FORBID();` but do not have any calls inside it.

Reply:
They are dead code too. Copy&paste or incomplete cleanup.